app/services/solr_mapper.rb

```diff
@@ -14,15 +14,85 @@ def initialize(metadata:, dataset_record_id:, dataset_record_set_id:)
   end
 
   # @return [Hash] the Solr document
+  # rubocop:disable Metrics/AbcSize
   def call
     {
       id: dataset_record_id,
-      dataset_record_set_id: dataset_record_set_id,
-      title: metadata.dig(:titles, 0, :title)
-    }
+      dataset_record_set_id_ss: dataset_record_set_id,
+      access_ssi: metadata['access'],
+      provider_ssi: metadata['provider'],
+      creators_struct_ss: metadata['creators'].to_json,
+      descriptions_tsim: retrieve_descriptions(metadata['descriptions'] || []),
```

Comment:
Can retrieve_descriptions just return the correct value instead of needing the `|| []` here? It reduces the cognitive complexity of reading this method.

Reply:
Will do